File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task62_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import org.json.JSONObject;

public class Main {
    public static void main(String[] args) {
        String jsonString = "{\"name\":\"John\", \"age\":30, \"city\":\"New York\"}";
        JSONObject jsonObject = new JSONObject(jsonString);
        System.out.println("Root element: " + jsonObject.keys().next());
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task62_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
// Note: This code uses the nlohmann/json library, which is a popular JSON library for C++.
// You need to install it before compiling the code.

#include <nlohmann/json.hpp>
#include <iostream>

using json = nlohmann::json;

int main() {
    std::string jsonString = "{\"name\":\"John\", \"age\":30, \"city\":\"New York\"}";
    json jsonObject = json::parse(jsonString);
    std::cout << jsonObject.type_name() << std::endl;
    return 0;
}